smart-contracts/contracts/cl-vault/src/vault/withdraw.rs

```diff
@@ -81,22 +76,19 @@ fn withdraw(
     let bq = BankQuerier::new(&deps.querier);
     let vault_denom = VAULT_DENOM.load(deps.storage)?;
 
-    let total_shares: Uint128 = bq
+    let total_vault_denom_amount: Uint128 = bq
```

Comment:
using vault_denom is more confusing than shares here, since the vault_denom is the shares for this vault

Reply:
would `total_vault_shares` be ok?